src/main/webapp/features/screens/game/dashboard/gameMap/ct-game-map.js

```diff
@@ -31,7 +43,29 @@ angular.module('catan')
                         var edgeId = angular.element(event.target).closest(DrawMapService.EDGE_SELECTOR).attr('edge-id');
                         SelectMapObjectService.select('edge', edgeId);
                     });
+
                 }
             };
 
+            //TODO: move that code to some helper?
+            function highlightHexes(element, map, dice, timeout) {
+                var hexesToHighlight = map.hexes.filter(function(hex) {
+                    return (dice === 7) ? hex.robbed : !hex.robbed && hex.dice === dice;
+                });
+
+                $timeout.cancel(timeout);
```

Comment:
if there is no timer, then we have nothing to cancel, so why do we need to pass timeout in highlightHexes( ) every time?